### PR TITLE
Added blitz.pc

### PR DESCRIPTION
**Pull request by AliKhoda**

### CODE REVIEW

File: recipe/bld.bat [hunk 1]
```diff
@@ -12,3 +12,38 @@ if errorlevel 1 exit 1
...
+copy %SRC_DIR%\blitz\*.h %LIBRARY_INC%\blitz\
+copy %SRC_DIR%\blitz\*.cc %LIBRARY_INC%\blitz\
+copy %SRC_DIR%\blitz\array\*.h %LIBRARY_INC%\blitz\array\
+copy %SRC_DIR%\blitz\array\*.cc %LIBRARY_INC%\blitz\array\
+copy %SRC_DIR%\blitz\gnu\*.h %LIBRARY_INC%\blitz\gnu\
+copy %SRC_DIR%\blitz\gnu\*.cc %LIBRARY_INC%\blitz\gnu\
+copy %SRC_DIR%\blitz\meta\*.h %LIBRARY_INC%\blitz\meta\
+copy %SRC_DIR%\blitz\meta\*.cc %LIBRARY_INC%\blitz\meta\
+copy %SRC_DIR%\blitz\ms\*.h %LIBRARY_INC%\blitz\ms\
+copy %SRC_DIR%\blitz\ms\*.cc %LIBRARY_INC%\blitz\ms\
```
Comment (183amir) [inline recipe/bld.bat]: Why are you copying the .cc files?
Also you need to bump the build number too to get a new package:
https://conda-forge.org/#update_recipe

File: recipe/meta.yaml [hunk 2]
```diff
@@ -36,6 +36,7 @@ test:
     - test -e $PREFIX/lib/libblitz.dylib   # [osx]
     - if not exist %PREFIX%\Library\bin\blitz.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\blitz.lib exit 1  # [win]
+    - if not exist %PREFIX%\Library\lib\pkgconfig\blitz.pc exit 1  # [win]
```
Comment (183amir) [inline recipe/meta.yaml]: You better test for a couple of header files too

File: recipe/bld.bat (hunk 1 above)
Comment (AliKhoda) [reply recipe/bld.bat]: Valid point.

File: recipe/meta.yaml (hunk 2 above)
Comment (AliKhoda) [reply recipe/meta.yaml]: There is little or no chance these may be missing, however, you are right, it never hurts to check.

**Pull request merged**